src/extension/gogoduck/src/main/java/au/org/emii/gogoduck/worker/GoGoDuckModule_cars_australia_weekly.java

```diff
@@ -0,0 +1,18 @@
+package au.org.emii.gogoduck.worker;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class GoGoDuckModule_cars_australia_weekly extends GoGoDuckModule_cars {
+    @Override
+    public URIList getUriList() throws GoGoDuckException {
+        URIList uriList = new URIList();
+        try {
+            uriList.add(new URI("/mnt/imos-t3/climatology/CARS/2009/eMII-product/CARS2009_Australia_weekly.nc"));
```

Comment:
No... And it should be! I'll do it. Now that I'm an expert with dependency injection ;)